c2rust-analyze/src/main.rs

```diff
@@ -361,6 +361,50 @@ impl<'tcx> Debug for AdtMetadataTable<'tcx> {
     }
 }
 
+fn label_rvalue_tys<'tcx>(acx: &mut AnalysisCtxt<'_, 'tcx>, mir: &Body<'tcx>) {
```

Comment:
This will conflict with #886.  Should I change #886 to run in a separate, second loop, or should we rename this function (given it'll label constant tys, too)?  The downside of two loops is then the match is separate into two places.

Reply:
I'm fine with it being renamed and updated to include code to label consts too. I don't think two loops is desirable, at least to me. For these reasons and what I mentioned [here](https://github.com/immunant/c2rust/pull/886#issuecomment-1509189413) I think this is why having one `rvalue_tys` dict in #886 is best. It's not a hill I'm willing to die on, and leave it to your judgement. We'll have to adjust to this one way or another and my primary concern is that it's correct and that it's clear to the reader what's going on.

Reply:
@spernsteiner just had a good suggestion for keeping the single `rvalue_tys` map and adding a `Vec` of locations that are constants as a way to iterate through them easily later, so I could try that.

Reply:
Sure, I'll be working on it today.

Reply:
@aneksteind, it's done in #886's 8665bc2d8cb5c78fc840da1b3bdeabf96f68af9f now.